.changeset/five-cameras-hammer.md

```diff
@@ -0,0 +1,10 @@
+---
+'@backstage/backend-common': patch
+'@backstage/integration': patch
```

Comment:
It's a breaking change for `integration`, I think the best thing to do here is to create a separate changeset for just the `integration` package, along with some pointers for how to migrate

Reply:
Thanks, done, Ive set it to minor. Hopefully that is correct.